### ISSUE TITLE
[QUESTION] Does khelm support oci helm registries?

### ISSUE THREAD

**Issue by milpog** [mgoltzsche/khelm#46]
Some projects like https://karpenter.sh are only publishing helm charts to oci. Those registries are supported from [Helm v3.8.0](https://helm.sh/docs/topics/registries/). I couldn't make khelm working with this type of registry.

- chart configuration
```yaml
apiVersion: khelm.mgoltzsche.github.com/v2
kind: ChartRenderer
metadata:
  name: karpenter
  namespace: kube-system
repository: "oci://public.ecr.aws/karpenter"
chart: karpenter
version: 0.19.2
```

- command to render helm chart
```bash
export KHELM_TRUST_ANY_REPO=true
kustomize build --enable-alpha-plugins . > karpenter yaml
```

- error message
```
Running khelm 2.2.1 (helm 3.10.1)
Using repository "oci://public.ecr.aws/karpenter" (unauthenticated)
Downloading repository index of oci://public.ecr.aws/karpenter
khelm: load chart karpenter: looks like "oci://public.ecr.aws/karpenter" is not a valid chart repository or cannot be reached: object required
```

**Comment by mgoltzsche** [comment]
@milpog thanks for creating the issue! I wasn't aware of the OCI registry support within helm. When Helm supports it, khelm should also support it. I wonder why it isn't working oob since khelm actually uses Helm. A pull request to also make this work within khelm would be appreciated.
Otherwise I may find some time to look into that myself at some point but currently I cannot spare time for this.

**Comment by husira** [comment]
@mgoltzsche Is OCI supported meanwhile or is this pending?

**Comment by mgoltzsche** [comment]
Oh, I am afraid this issue was de-prioritized. OCI repos are still not supported by the latest khelm release. Though, I almost finished implementing that feature within #55. I'll finish and merge that PR the next days...

**Comment by mgoltzsche** [comment]
Now this is fixed within [khelm v2.3.2](https://github.com/mgoltzsche/khelm/releases/tag/v2.3.2). Sorry for the long delay :sweat_smile:.

**Comment by husira** [comment]
Thank you very much for the information and for implementing OCI support :)

**Comment by mgoltzsche** [comment]
Using this feature myself really I realized it wasn't working correctly. I [fixed it now](https://github.com/mgoltzsche/khelm/pull/75) and released the fixes with [v2.3.4](https://github.com/mgoltzsche/khelm/releases/tag/v2.3.4).
